src/pubsub.c

```diff
@@ -722,3 +722,15 @@ void sunsubscribeCommand(client *c) {
     }
     if (clientTotalPubSubSubscriptionCount(c) == 0) c->flags &= ~CLIENT_PUBSUB;
 }
+
+size_t pubsubMemOverhead(client *c) {
+    // PubSub patterns
```

Comment:
please avoid line comments.
one of the few conventions in redis which is consistent is that we don't use `//`.